src/test/java/com/devonfw/sample/archunit/ComponentRuleTest.java

```diff
@@ -13,7 +13,7 @@
 import com.tngtech.archunit.lang.ConditionEvents;
 import com.tngtech.archunit.lang.SimpleConditionEvent;
 
-@AnalyzeClasses(packages = "com.devonfw.sample.archunit", importOptions = ImportOption.DoNotIncludeTests.class)
+@AnalyzeClasses(packages = {"com.devonfw.sample.archunit","com.devonfw.sample.batch"}, importOptions = ImportOption.DoNotIncludeTests.class)
```

Comment:
I am not getting the intention of this change.
`com.devonfw.sample.archunit` is the root package of our sample appliction.
Below it we follow the devonfw package convention with `«component».«layer»`.
In the `task` component, you could simply add a package `batch` for potential batch code and according violation code without this change.

Reply:
that was a misunderstanding from my side. I will close the pull request. I thought the batch has to be external from our archunit structure.